src/components/table/Table.js

```diff
@@ -1301,6 +1305,13 @@ class Table extends Component {
             handleToggleExpand={toggleFullScreen}
           />
         )}
+        {isMobileOrTablet && rows.length > MOBILE_TABLE_SIZE_LIMIT && (
+          <span className="text-danger">
+            {/*TODO: use here translations, but how we define new keys?*/}
```

Comment:
@ionut-movila ping @teosarca or @metas-ts and ask them to add whatever translations you need

Reply:
introduced `view.limitTo`.
See https://w101.metasfresh.com:8443/rest/api/i18n/messages?filter=view.limitTo for current value.

Reply:
I have fixed this with commit: https://github.com/metasfresh/metasfresh-webui-frontend/pull/2290/commits/ccd7caaf1b0d7470acb1c6a816fadd4ec411196f

@metas-ts But we need to update the string from backend with respect to interpolation rules of the FE lib: https://www.npmjs.com/package/counterpart#interpolation

So it should be like this: "Limited to %(limit)s rows of %(total)s"

Reply:
done